src/main/java/frc/robot/commands/GetCenterNoteX.java

```diff
@@ -52,11 +51,12 @@ public GetCenterNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVisio
             ( new DeferredCommand(() -> m_driveTrain.followPath(getInitialPath()), Set.of(m_driveTrain))
                 .deadlineWith(new MonitorForNote(noteVision, () -> m_driveTrain.getPose(), m_targetNote, this))
             ).deadlineWith(
-                new WaitUntilCommand(m_driveTrain::isInCenterZone).andThen(new StartIntake(intake, shooter, shooterPivot, elevator))
+                // new WaitUntilCommand(m_driveTrain::isInCenterZone).andThen(new StartIntake(intake, shooter, shooterPivot, elevator))
+                new StartIntake(intake, shooter, shooterPivot, elevator)
             ),
 
             // wait up to 1 second to suck the Note in all the way
-            // new WaitUntilCommand(intake::hasNote).withTimeout(1),
+            // new WaitUntilCommand(intake::hasNote).withTimeout(INTAKE_EXTRA_WAIT_TIME),
             new WaitCommand(INTAKE_EXTRA_WAIT_TIME),
 
             // turn off Shooter and intake
```

Comment:
Not for this PR, but as a next step:  we should check here for Intake.hasNote() -- and only continue on return path if true, finish early otherwise.

Reply:
Agreed. We should check log files to see if we think it is working.